### ISSUE TITLE
transparent color not recognized

### ISSUE THREAD

**Issue by Frikki** [typestyle/typestyle#25]
- typestyle v0.5.6

This code doesn’t render transparent:

``` javascript
const myStyle = style({
  color: 'transparent'
})
```

However, this does:

``` javascript
const myStyle = style({
  color: 'rgba(0, 0, 0, 0)'
})
```

**Comment by notoriousb1t** [comment]
I added a unit test to check that it is outputting transparent for color: 

```typescript
    cssRule('.transparent', { color: 'transparent' });
    style({ color: 'transparent' });
    assert.equal(css(), '.transparent{color:transparent}.fwarpl0{color:transparent}');
```

Which browsers is this affecting?  Can you inspect your web application and let us know what value the color property has in the browser?

**Comment by Frikki** [comment]
My bad! It is recognized. I can’t really tell why it wasn’t yesterday; now it is. Sorry for the inconvenience.

**Comment by notoriousb1t** [comment]
Not a problem at all :)

As an aside, we just released 0.6.0 today which has a whole bunch color helpers included:

```typescript
import { rgb, hsla, transparent } 'typescript/csx';

const primary = rgb(0, 0, 255).fadeOut('10%');
const secondary = hsla(240, .5, .5, 1).darken('10%');
const ternary = transparent; // rgb(0, 0, 0, 0)
```

You should check them out!
